src/DataViews/FunctionsDataViewTest.cpp

```diff
@@ -357,23 +358,6 @@ TEST_F(FunctionsDataViewTest, AddressColumnShowsAddress) {
   EXPECT_EQ(view_.GetValue(0, 4).substr(2), absl::StrFormat("%x", functions_[0].address()));
 }
 
-TEST_F(FunctionsDataViewTest, CommonContextMenuEntriesArePresent) {
```

Comment:
Why this is not needed anymore?

Reply:
It should be sufficient to have the `ContextMenuEntriesChangeOnFunctionState` test.

I refactored the `ContextMenuEntriesChangeOnFunctionState` to cover all the test cases in some of my previous PR but forgot to remove this redundant test.